packages/frontend/src/Build.spec.ts

```diff
@@ -80,6 +80,79 @@ const mockBootcImages: ImageInfo[] = [
   },
 ];
 
+const mockImageInspect: ImageInspectInfo = {
```

Comment:
There is so much boilerplate/default values here I am wondering if it is a case to remove most of it and use `as unknown as ImageInspectInfo` :)